lib/api/model/initial_snapshot.dart

```diff
@@ -68,6 +68,8 @@ class InitialSnapshot {
 
   final int maxFileUploadSizeMib;
 
+  final Uri serverEmojiDataUrl;
```

Comment:
Yeah, probably best to handle that explicitly. We can skip actually trying to provide data for those servers (since they're already out of the 18-month support window), though.